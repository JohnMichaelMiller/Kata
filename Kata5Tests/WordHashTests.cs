using Microsoft.VisualStudio.TestTools.UnitTesting;
using Kata5;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using ApprovalTests;
using ApprovalTests.Reporters;
using ApprovalUtilities.Utilities;

namespace Kata5.Tests
{
    [TestClass()]
    [UseReporter(typeof(DiffReporter))]
    public class WordHashTests
    {
        [TestMethod()]
        [UseReporter(typeof(ClipboardReporter), typeof(DiffReporter))]
        public void WordHashesTest()
        {
            var words = new WordHash(@"C:\Users\John\Source\Repos\Kata\Kata5\wordlist.txt");
            Approvals.VerifyAll(words, d => "Key {0}, Hash Length {1}".FormatWith(d.Key, d.Value.Length));
        }
    }
}