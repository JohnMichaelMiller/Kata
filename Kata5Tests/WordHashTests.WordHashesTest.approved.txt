Key A, Hash Length 16
Key A'asia, Hash Length 16
Key A's, Hash Length 16
Key AA, Hash Length 16
